evm_loader/program/src/executor/state.rs

```diff
@@ -431,13 +433,15 @@ impl<'a, B: AccountStorage> Database for ExecutorState<'a, B> {
         Ok(cache.block_timestamp)
     }
 
+    #[allow(clippy::too_many_lines)]
```

Comment:
You don't need it here.
If you think this function is too long, split it into several smaller.